WcaOnRails/db/seeds/development/competitions.seeds.rb

```diff
@@ -153,32 +156,17 @@ def random_wca_value
       latitude_degrees: rand(-90.0..90.0),
       longitude_degrees: rand(-180.0..180.0),
     )
+    events = random_events
+    competition.events = events
```

Comment:
I think it's ok to do `competition.events.each ...` and we no longer need this variable, but follow your opinion here =)